### PR TITLE
Add current station time to game lobby

### PR DESCRIPTION
**Pull request by jessicamaybe**
## About the PR 

Adds current station time to the lobby screen. It's useful for late joiners so they get an idea of how long the round has been going on. Also doesn't update unless the rest of the text is updated but I don't really know if it needs to be.

**Screenshots**

image

**Changelog**

:cl:
- add: You can now see the station time in the lobby screen

### CODE REVIEW

File: Content.Server/GameTicking/GameTicker.Lobby.cs [hunk 1]
```diff
@@ -45,7 +45,8 @@ private string GetInfoText()
             var mapName = map?.MapName ?? Loc.GetString("game-ticker-no-map-selected");
             var gmTitle = Loc.GetString(Preset.ModeTitle);
             var desc = Loc.GetString(Preset.Description);
-            return Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc));
+            var roundTime = _gameTiming.RealTime.ToString("hh\\:mm");
+            return Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName), ("roundTime", roundTime), ("gmTitle", gmTitle),("desc", desc));
```
Comment (metalgearsloth) [inline Content.Server/GameTicking/GameTicker.Lobby.cs:49]: Is RealTime reset on round restart? I didn't think content touched any of the engine times.

Comment (jessicamaybe) [reply Content.Server/GameTicking/GameTicker.Lobby.cs:49]: Looks like it doesn't, I'll just change it to RoundDuration

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
This should still update even if i'm in the lobby but right now it's only refreshed once by the server when the client joins and never again.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
This still has timing issues where the start time is being set past RealTime which means the timer starts at 1 minute, goes back down to 0 minutes, then starts going up again.

**Comment by jessicamaybe** [comment]
> This still has timing issues where the start time is being set past RealTime which means the timer starts at 1 minute, goes back down to 0 minutes, then starts going up again.

well looks like starttime did not do what I thought it did, I switched it to use _roundStartTimeSpan and it seems to be working properly now even when the round starts early or is delayed.

File: Resources/Locale/en-US/lobby/lobby-state.ftl [hunk 2]
```diff
@@ -6,4 +6,5 @@ lobby-state-ready-button-join-state = Join
 lobby-state-ready-button-ready-up-state = Ready Up
 lobby-state-player-status-not-ready = Not Ready
 lobby-state-player-status-ready = Ready
-lobby-state-player-status-observer = Observer
\ No newline at end of file
+lobby-state-player-status-observer = Observer
+lobby-state-player-status-station-time =  The Station Time is: {$stationTime}
```
Comment (mirrorcult) [inline Resources/Locale/en-US/lobby/lobby-state.ftl:10]:
```suggestion
lobby-state-player-status-station-time =  The station time is: {$stationTime}
```

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**